app/views/layouts/messages.html.erb

```diff
@@ -3,18 +3,17 @@
 
   <div class="flex h-screen"
       data-controller="click-toggle"
-      data-click-toggle-flippable-class="md:flex"
-      data-action="keydown.meta+shift+s@document->click-toggle#handleEvent
-                   keydown.meta+period@document->click-toggle#handleEvent"
+      data-click-toggle-flippable-class="!hidden"
   >
 
     <div id="left-column"
-        class="flex-col h-screen w-[260px] bg-black hidden md:block overflow-y-auto overflow-x-clip scrollbar-hide"
+        class="flex-col h-screen w-[260px] bg-black hidden md:flex overflow-y-auto overflow-x-clip scrollbar-hide"
         data-click-toggle-target="destination"
     >
 
       <div id="left-content-container" class="flex flex-col flex-grow">
-        <div class="pl-3 pr-5 flex flex-col mb-28">
+        <div class="pl-3 flex flex-col mb-28">
+          <div id="spacer-above-sticky-assistant" class="sticky top-0 h-4 z-10 bg-black"></div>
```

Comment:
I'm generally surprised any time a z-index is needed, but in this case I'm a little more surprised.

Reply:
I believe "sticky" ensures that the subsequent items scroll underneath it, however all those items have tooltips and the tooltips have a z-index so they were floating above this sticky header as the items were scrolling underneath it. This ensures the header is above any tooltips.